src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -179,6 +183,18 @@ class TextGenerationOutput(BaseModel):
         "prompt provided. If streamng is enabled, the next generated token is returned."
         "Otherwise, the full generated sequence is returned."
     )
+    input_tokens: Optional[
+        Any
+    ] = Field(  # dictionary mapping "token_ids" and "attention_mask" to numpy arrays
+        default=None,
+        description="The output of the tokenizer."
+        "Dictionary containing token_ids and attention_mask, "
+        "both mapping to arrays of size "
+        "[batch_size, sequence_length]",
+    )
+    session_id: Optional[str] = Field(
```

Comment:
need rebase. `session_id` is no longer part of this class.

Reply:
Fixed